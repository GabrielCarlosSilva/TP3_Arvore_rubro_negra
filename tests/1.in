1
2
Luan
10
Ana 
24
1
3
Lucas
23
Luisa
29
Laura 
13
0
