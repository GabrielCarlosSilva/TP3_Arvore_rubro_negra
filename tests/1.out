Dados inOrder:
Nome: Luan
Idade: 10
Nome: Laura
Idade: 13
Nome: Lucas
Idade: 23
Nome: Ana
Idade: 24
Nome: Luisa
Idade: 29